fix bug with the docker-compose

xzxiaoshan opened the pull request:
fix bug：docker compose >>>> docker-compose

nabokihms commented:
Hello, @xzxiaoshan. Actually, this is not a bug. We intend your to use the compose plugin https://docs.docker.com/compose/install/linux/, which is a subcommand of a docker.

xzxiaoshan commented:
@nabokihms 

"Now I understand the Docker Compose plugin mode for Docker. I believe it's important to at least add a comment explaining this, as I've noticed that other users have also raised this issue in their reports. Of course, the author can consider whether to default to using docker-compose or docker compose. For now, I have reverted my changes and added an explanation. I think this clarification will be helpful to some users, as it can at least eliminate confusion."

thks.

xzxiaoshan commented:
@nabokihms 

I don't know if it's possible to consider using my script below to be compatible with either of the two methods. The example is as follows:

```bash
#!/bin/bash

# Function to check if docker-compose is available
check_docker_compose() {
  if command -v docker-compose >/dev/null 2>&1; then
    echo "Found standalone docker-compose."
    DOCKER_COMPOSE_CMD="docker-compose"
  elif docker compose version >/dev/null 2>&1; then
    echo "Found docker compose plugin."
    DOCKER_COMPOSE_CMD="docker compose"
  else
    echo "Neither docker-compose nor docker compose plugin found. Please install one of them."
    exit 1
  fi
}

# Check for Docker Compose availability
check_docker_compose

# Start services using the detected Docker Compose command
$DOCKER_COMPOSE_CMD up -d

# Additional commands can be added here
$DOCKER_COMPOSE_CMD ps

echo "Docker Compose service started successfully using: $DOCKER_COMPOSE_CMD"
```

nabokihms commented:
@xzxiaoshan It seems like commenting is good enough. Thank you for your contribution.
